scipy/stats/tests/test_continued_fraction.py

```diff
@@ -14,6 +14,7 @@
 # n = int(xp.real(xp_ravel(n))[0])
 # (at some point in here the shape becomes nan)
 @pytest.mark.skip_xp_backends('dask.array', reason="dask has issues with the shapes")
+@pytest.mark.uses_xp_capabilities(False, reason="private")
```

Comment:
I think it would be about as easy to use `xp_capabilities`, unless private functions are not supposed to.